### DISCUSSION TITLE
vcpkg

### DISCUSSION THREAD

**Discussion by davidhunter22** [GabrielDosReis/ipr#256] [Ideas]
Add a vcpkg for library consumers.
I can take a look at doing this if you are OK with it although I am no expert in how to do it.

**Comment by GabrielDosReis** [comment]
I would not be able to keep up with updating the vcpkg port in addition to the mods I am doing here.  So, if you aren't creating additional work for me, I'm fine with that...

**Comment by davidhunter22** [comment]
I will have a try packaging it. I assume it will be just Cmake changes. Then you can take a look.
